thirdparty/protobuf/CMakeLists.txt

```diff
@@ -0,0 +1,125 @@
+# Copyright (C) 2018-2021 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+#
+
+#------------------------------------------------------------------------------
+# Configure and install Google Protobuf ...
+#------------------------------------------------------------------------------
+
+set(PUSH_CMAKE_INTERPROCEDURAL_OPTIMIZATION_RELEASE "${CMAKE_INTERPROCEDURAL_OPTIMIZATION_RELEASE}")
```

Comment:
since this file is used as `add_subdirectory` which introduces a new scope of variables, we can remove `PUSH_CMAKE_INTERPROCEDURAL_OPTIMIZATION_RELEASE` and set `set(CMAKE_INTERPROCEDURAL_OPTIMIZATION_RELEASE OFF)` directly and it does not affect global scope.